src/services/hmwk-service.js

```diff
@@ -1,90 +1,96 @@
-const initMondayClient = require('monday-sdk-js');
-
+const initMondayClient = require("monday-sdk-js");
 
 class HmwkService {
-
     static async getAllStudents(token, studentsBoardId) {
         try {
             const mondayClient = initMondayClient();
             mondayClient.setToken(token);
 
             const query = `query {
                 boards(ids: $studentsBoardId) {
-                  items {
-                    name
-              
-                    column_values {
-                      value
-                      text
+                    items {
+                        name
+            
+                        column_values {
+                            value
+                            text
+                        }
                     }
-                  }
                 }
-              }`
+            }`;
 
             const variables = { studentsBoardId };
 
             const response = await mondayClient.api(query, { variables });
-            const studentInfo = response.data.boards.items
-            return studentInfo //array of json
-            //student name: student_info[i].name 
+            const studentInfo = response.data.boards.items;
+            return studentInfo; //array of json
+            //student name: student_info[i].name
             //student email: student_info[i].column_values.text
-
         } catch (err) {
             console.log(err);
         }
```

Comment:
I know the text-transformation example does this but...

I think it's better to allow the caller of `getAllStudents()` decide how to handle the error, instead of catching it and printing it here. For example, perhaps we would want to respond with 500 status + an error msg